homeassistant/components/deconz/sensor.py

```diff
@@ -92,13 +92,17 @@ class DeconzSensorDescription(
     AirQuality: [
         DeconzSensorDescription(
             key="air_quality",
-            value_fn=lambda device: device.air_quality,  # type: ignore[no-any-return]
+            value_fn=lambda device: device.air_quality
```

Comment:
Please use a normal function instead of a lambda if the lambda spans more than one line.

Reply:
Thanks Martin! I will look into this shortly

Reply:
Finally resolved.